src/campsite-info/components/camp-rating/camp-rating.styles.ts

```diff
@@ -0,0 +1,10 @@
+export const ratingStyles = {
+  width: 200,
+  display: "flex",
+  alignItems: "center",
+}
+
+export const tooltipStyles = { 
+  ml: 0,
```

Comment:
let's stay away from the sx short forms. They're neat but I think they hurt readability.

Reply:
the latest push makes this change